src/msgmle.cpp

```diff
@@ -520,13 +521,47 @@ void ChatEdit::setEditText(const QString& text)
 
 void ChatEdit::insertFromMimeData(const QMimeData *source)
 {
+    auto obtainSourceText = [source](){
+        if (!source->text().isEmpty())
+            return source->text();
+        else
+            return QString(source->data("text/plain"));
```

Comment:
QString by default does fromUtf8. Are you sure it's always utf8 there? If it's in local encoding then you have to do QString::fromLocal8Bit

Reply:
Fixed